package.json

```diff
@@ -41,6 +41,7 @@
     "version": "npm run update-packages && git add greenkeeper.json \"**/package-lock.json\" && npm run update-template-deps && npm run apidocs",
     "outdated": "npm outdated --depth 0 && lerna exec --no-bail \"npm outdated --depth 0\"",
     "apidocs": "node bin/run-lerna run build:apidocs",
+    "tsdocs": "lerna run --scope @loopback/tsdocs build:tsdocs",
```

Comment:
do we still need command `apidocs` at the above line?

Reply:
We keep existing strong-docs based apidocs for now. The PR will allow new styles to be seen on loopback.io. Once we are happy with the new approach, we'll remove strong-docs.